articles/hdinsight/kafka/apache-kafka-producer-consumer-api.md

```diff
@@ -109,7 +109,17 @@ In this code, the consumer is configured to read from the start of the topic (`a
 
 The [Run.java](https://github.com/Azure-Samples/hdinsight-kafka-java-get-started/blob/master/Producer-Consumer/src/main/java/com/microsoft/example/Run.java) file provides a command-line interface that runs either the producer or consumer code. You must provide the Kafka broker host information as a parameter. You can optionally include a group ID value, which is used by the consumer process. If you create multiple consumer instances using the same group ID, they'll load balance reading from the topic.
 
-## Build and deploy the example
+## Use Pre-built JAR files
+
+Download the jars from the [Kafka Get Started Azure sample](https://github.com/Azure-Samples/hdinsight-kafka-java-get-started/tree/master/Prebuilt-Jars). If your cluster is **Enterprise Security Package (ESP)** enabled, use kafka-producer-consumer-esp.jar. Use the command below to copy the jars to your cluster.
+
+
+    ```cmd
+    scp kafka-producer-consumer*.jar sshuser@CLUSTERNAME-ssh.azurehdinsight.net:kafka-producer-consumer.jar

```

Comment:
Hey Hassan, help here   its not rendering correctly when I preview

Reply:
Try using 3 spaces indentation instead of 4 spaces. 4-spaces indicates code, which is conflicting with ```cmd.